earthpy/tests/test_spatial.py

```diff
@@ -18,6 +18,19 @@ def b1_b2_arrs():
     return b1, b2
 
 
+@pytest.fixture
+def hillshade_arr():
```

Comment:
Is the intent of this fixture to create a 3-dimensional array? it might be good to add a docstring to clarify that potentially? i just noticed you using squeeze below and wondered if the intent was this was 3-dims and so you could test what happened if a 3 dim array was provided to the function.

Reply:
That is correct. I've added a docstring to this fixture to make that clear.